vault/diagnose/os_common.go

```diff
@@ -26,16 +26,18 @@ partLoop:
 			}
 		}
 		usage, err := disk.Usage(partition.Mountpoint)
-		testName := "disk usage"
+		testName := "Disk Usage"
```

Comment:
Noun to verb suggestion: "Calculate Disk Usage"

Reply:
Used "Check Disk Usage" to keep the prefix normative.